content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -86,8 +86,10 @@ model Tag {
 }
 ```
 
-- If you delete a `Tag`, the corresponding tag assignment is also deleted `TagOnPost` (`Cascade`)
-- If you delete a `User`, the author is removed from all posts (`SetNull`) - `User` and `userId` must be optional
+This model explicitly defines the following referential actions:
+
+- If you delete a `Tag`, the corresponding tag assignment is also deleted in `TagOnPosts`, using the `Cascade` referential action
+- If you delete a `User`, the author is removed from all posts, using the `SetNull` referential action. To allow this, `User` and `userId` must be optional fields in `Post`.
```

Comment:
```suggestion
- If you delete a `Tag`, the corresponding tag assignment is also deleted in `TagOnPosts`, because of the `Cascade` referential action
- If you delete a `User`, the author is removed from all posts by setting the field value to `Null`, because of the `SetNull` referential action. To allow this, `User` and `userId` must be optional fields in `Post`.
```

Reply:
Done as part of other changes